onadata/settings/default_settings.py

```diff
@@ -10,7 +10,7 @@
 from onadata.settings.staging_example import *  # noqa
 
 
-ALLOWED_HOSTS = ['*']
+ALLOWED_HOSTS = ['*', '0.0.0.0']
```

Comment:
Codacy found an issue: [Possible binding to all interfaces.](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5740788)